### ISSUE TITLE
Broken aria-describedby reference on OverlayTrigger to tooltip element

### ISSUE THREAD

**Issue by tragessere** [react-bootstrap/react-bootstrap#4934]
## Describe the bug

The OverlayTrigger adds an `aria-desribedby` attribute to it's target when showing (if the popper has a tooltip role). When the tooltip is no longer showing the `aria-describedby` attribute is left

## To Reproduce

1. Create an OverlayTrigger with an overlay that has the 'tooltip' role and and id. Add an element inside the OverlayTrigger to activate it such as a button (any trigger type is fine).
2. The element correctly does not have an `aria-describedby` attribute at the start.
2. Activate the popover.
3. The element now correctly has an `aria-describedby` attribute referencing the popover.
4. Close the popover.
5. The element now incorrectly still has the `aria-describedby` attribute which no longer references anything in the DOM.

## Reproducible Example

Go to https://react-bootstrap.github.io/components/overlays/#popover-examples. Look at the attributes for the 'Click me to see' example button before showing the popover and after closing it.

## Expected behavior

The `aria-describedby` attribute should be removed from the OverlayTrigger's target element when not showing the popover. Alternatively the popover could be left in the DOM and shown/hid dynamically instead and the `aria-describedby` attribute could always be on the target element.

## Environment (please complete the following information)

- Operating System: Windows
- Browser, Version Chrome 79
- React-Bootstrap Version 1.0.0-beta.16

## Additional context

The dynamic `aria-describedby` attribute was added in a merge request as part of this old issue: https://github.com/react-bootstrap/react-bootstrap/issues/1827

**Comment by taion** [comment]
Looks like we're missing a cleanup function for the ARIA modifier. We'd be happy to take a PR to address this.

**Comment by ocBruno** [comment]
A little new to this environement but i'm going to give this one a shot, if anyone already working on this give me a heads up!

**Comment by tragessere** [comment]
Closing this issue. It was fixed as part of https://github.com/react-bootstrap/react-bootstrap/pull/5017
